src/components/Error/ErrorNotification.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  hasError: boolean,
+  setHasError: (value: boolean) => void,
```

Comment:
```suggestion
  setHasError: React.Dispatch<React.SetStateAction<Boolean>>,
```
Use more specific types fix it everywhere